### ISSUE TITLE
"Go To Definition" unnecessarily shows import definition when using jsconfig.json

### ISSUE THREAD

**Issue by sefranz** [microsoft/vscode#137111]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.62.1
- OS Version: Windows 11 Home (21H2)

Steps to Reproduce:

1. Clone [this](https://github.com/sefranz/vs-nav-test) minimal example
2. Open file `/src/pages/test.jsx`
3. Focus the jsx-tag `<HelloWorlder />`
4. Invoke "Go To Definition" (Ctrl+Click / via context menu / F12)
5. The focus jumps to the import statement in line 2

This behavior might be correct, but highly useless. This way, you have to always manually focus the imported filename and invoke the command again in order to open the corresponding file.

Also, this behavior does not occur when deleting the `jsconfig.json` file.

![nav-test](https://user-images.githubusercontent.com/44921311/141642105-34e00f5c-6c0b-4d58-8126-87d2c428c6f1.gif)

**Comment by mjbvz** [comment]
Does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/)?

**Comment by sefranz** [comment]
Yes, it does in 1.63.0-insider (commit 035eaab). As far as I can distinguish, this problem seems to be limited to JSX-Code, though.
